virtual_ecosystem/data_variables.toml

```diff
@@ -215,6 +215,13 @@ name = "layer_heights"
 unit = "m"
 variable_type = "float"
 
+[[variable]]
+axis = ["spatial"]
+description = "Interception"
```

Comment:
Think this needs a longer description

Reply:
Oh hell, they _all_ need a longer description. And units. And proper axes. I think we're gunna need a bigger PR 😄